src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -107,6 +108,7 @@ const ApplicationDetails: React.FC = () => {
     "approved" | "rejected" | "resubmit"
   >();
   const [criteriaResults, setCriteriaResults] = useState<any[]>([]);
+  const [fullApplicationData, setFullApplicationData] = useState<any>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)